plugins/inventory/ovirt.py

```diff
@@ -249,6 +245,14 @@ def parse(self, inventory, loader, path, cache=True):
         self.ovirt_engine_user = self.get_option('ovirt_username')
         self.ovirt_engine_password = self.get_option('ovirt_password')
         self.ovirt_engine_cafile = self.get_option('ovirt_cafile')
+        self.ovirt_insecure = self.get_option('ovirt_insecure')
+        self.connection = sdk.Connection(
+            url=self.ovirt_engine_url,
+            username=self.ovirt_engine_user,
+            password=self.ovirt_engine_password,
+            ca_file=self.ovirt_engine_cafile,
+            insecure=self.ovirt_insecure,
```

Comment:
Don't create self.ovirt_engine_user instead, parse it to connection Connection because the self.ovirt_engine_user is not used anywhere else. This is for all attributes.